src/server/api/routers/notebook.ts

```diff
@@ -12,13 +12,20 @@ export const notebookRouter = createTRPCRouter({
   update: publicProcedure
     .input(z.object({ widgetId: z.string(), content: z.string(), configName: z.string() }))
     .mutation(async ({ input }) => {
+      if (!process.env.DISABLE_EDIT_MODE) {
```

Comment:
This env variable will not be present in the upcoming update. So we need to use the session then